### ISSUE TITLE
[Bug] [a11y] Automatic theme is not read

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#3921]
### Prerequisites
Make sure you have a clean profile

### Steps to reproduce
1. Enable Talkback on your device.
2. Swipe through Onboarding until you reach "Choose your theme" card - Automatic.

### Expected behavior
The Automatic theme should be read.

### Actual behavior
The automatic theme is not read.

### Device information
* Android device: Motorola Moto G6 (Android 8), Sony Xperia Z2 tablet (Android 6.0.1)
* Fenix version: 1.0.1 RC build 1

**Comment by mcarare** [comment]
@softvision-miralobontiu can you please check if this is still an issue?

**Comment by softvision-miralobontiu** [comment]
It is not reproducible anymore on RC 1.3.0-rc.3 on Xiaomi Mi8 Lite (Android 9).
